megamek/src/megamek/common/Infantry.java

```diff
@@ -73,6 +73,8 @@
             | FIRE_ENGINEERS | MINE_ENGINEERS | SENSOR_ENGINEERS
             | TRENCH_ENGINEERS;
 
+    public static final double PRIMARY_WEAPON_DAMAGE_CAP = 0.6;
```

Comment:
Can we have this in MegaMekConstants instead? Would be nice to start grouping our constants together there.

Reply:
This constant has no use outside this class, and this is the logical place for it.

Reply:
It is isolated here, but I'm thinking about the far future. ​If we have all of our constants isolated to a single file, we could eventually do a constant definition file for significantly increased user customizability.